lib/cc/engine/analyzers/php/parser.rb

```diff
@@ -16,7 +16,7 @@ def initialize(code, filename)
           end
 
           def parse
-            runner = CommandLineRunner.new("php #{parser_path}")
+            runner = CommandLineRunner.new("php -d 'display_errors = Off' #{parser_path}")
```

Comment:
This looks like a good bugfix in its own right, and not directly related to the new feature. It's already in its own commit, so I'd support cherry-picking this into a new branch & shipping it separately as its own PR if you want to. 

Also, it's been many years since I wrote much PHP, so pardon if this is a silly question: this won't stop us from seeing exceptions/stack traces in logs if the parser crashes, will it?